src/nft/hooks/useIsMobile.ts

```diff
@@ -0,0 +1,26 @@
+/* tslint:disable */
```

Comment:
This is no longer needed. The global app uses `import { isMobile } from 'utils/userAgent'`

Reply:
The functionality is fundamentally different between the two different hooks. The nft useIsMobile hook is dependent on the screenwidth breakpoints set by design whereas the userAgent isMobile hook just checks if a user's device is a tablet or mobile device